core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
 | Databricks AWS   | 8x m6gd.8xlage    | 8x g5.8xlarge                  |
 | Databricks Azure | 8x E8ds_v4        | 8x NC8as_T4_v3                 |
 
-Note that all benchmarks were run using the [NDS benchmark](https://github.com/NVIDIA/spark-rapids-benchmarks/tree/dev/nds) at SF3K (3 TB).
-
 > **Disclaimer!**  
-> Estimates provided by the Qualification tool are based on the currently supported "_SparkPlan_" or "_Executor Nodes_"
-> used in the application. It currently does not handle all the expressions or datatypes used.  
-> Please refer to "[Understanding Execs report](#execs-report)" section and the
-> "[Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md)" guide to check the types and expressions you are using are supported.
+> Estimates provided by the Qualification tool assume the application is run on a dedicated cluster where it can use all available Spark resources. The estimates are based on the "_SparkPlan_" or "_Executor Nodes_"
+> used in the Spark application and currently supported by the RAPIDS Accelerator for Apache Spark. Not all expressions and datatypes are currently supported,
```

Comment:
At some point, @mattahrens went through the entire documentation to unify our naming-reference to both "_Spark_" and "_plugin_".
So, "Spark application" -> "Apache Spark application".
This is applied to the entire files